CITATION.cff

```diff
@@ -0,0 +1,27 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Cordier"
+  given-names: "Thibault"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+title: "MAPIE - Model Agnostic Prediction Interval Estimator"
+version: 0.8.3
+date-released: 2019-04-30
+url: "https://github.com/scikit-learn-contrib/MAPIE"
+preferred-citation:
+  type: article
+  authors:
+  - family-names: "Taquet"
+    given-names: "Vianney"
+  - family-names: "Blot"
+    given-names: "Vincent"
+  - family-names: "Morzadec"
+    given-names: "Thomas"
+  - family-names: "Lacombe"
+    given-names: "Louis"
+  - family-names: "Brunel"
+    given-names: "Nicolas"
+  doi: "10.48550/arXiv.2207.12274"
+  journal: "arXiv preprint arXiv:2207.12274"
+  title: "MAPIE: an open-source library for distribution-free uncertainty quantification"
```

Comment:
Is it possible to quote the paper presented to COPA (as a replacement or in addition)?

Reply:
Could you remind me the link to this paper? My only concern is that this paper also presented a new insight and was not a general overview of the library.

What do you think @vincentblot28 ?

Reply:
I think we must add the COPA paper as we also present MAPIE in it. https://proceedings.mlr.press/v204/cordier23a/cordier23a.pdf